### ISSUE TITLE
Make ratings widget keyboard accessible

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#1360]
### Describe the problem and steps to reproduce it:

- Visit https://amo.addons-dev.allizom.org/en-US/firefox/addon/adblock-plus/ 
- Using just the keyboard navigate to the ratings widget
- Select a rating star using the keyboard
- try and submit the rating

### What happened?

- Focus styles are shown and it's possible to tab to the different stars 
- The stars could change to yellow as they are focused along with the previous stars to provide a clearer visual indication of what submission would do.
- It's not possible to submit a rating without a click.

### What did you expect to happen?

- It should be possible to clearly navigate the widget and submit a rating without a mouse.

### Anything else we should know?

<img  alt="add-ons_for_firefox" src="https://cloud.githubusercontent.com/assets/1514/20171971/1113ef36-a72b-11e6-895e-43109131008d.png">

**Comment by muffinresearch** [comment]
Turns out the submission works fine, I should have checked the request and would have seen a 401 because I'm not logged in!

![shamecube](https://cloud.githubusercontent.com/assets/1514/20188275/e2485c7e-a76f-11e6-9cef-48eeffbaa891.gif)

It just appeared to do nothing because there's not yet any error handling. So fixing this is just about the minor issue of updating the stars on focus.

**Comment by kumar303** [comment]
hah, well, hiding errors is pretty much the worst UI behavior ever. It should be fixed in https://github.com/mozilla/addons-frontend/issues/1032

**Comment by vcarciu** [comment]
I am not sure if this still needs qa and if yes, can you add some STRs?

**Comment by tofumatt** [comment]
I don't think this still needs QA; the keyboard should work and submit a rating, but I think that's covered elsewhere.
